src/objective-c/tests/CronetUnitTests/CronetUnitTests.m

```diff
@@ -270,8 +270,11 @@ - (void)testInternalError {
 }
 
 - (void)PacketCoalescing:(bool)use_coalescing {
-  grpc_cronet_use_packet_coalescing(use_coalescing);
-
+  grpc_arg arg;
+  arg.key = GRPC_ARG_USE_CRONET_PACKET_COALESCING;
+  arg.type = GRPC_ARG_INTEGER;
+  arg.value.integer = use_coalescing ? 1:0;
+  grpc_channel_args *args = grpc_channel_args_copy_and_add(NULL, &arg, 1);
```

Comment:
Since things being tested here are core changes, I would rather not to invoke something in ObjC library. Besides that I feel it is a bit over weight to move that code here and create `grpc_channel_args` with it; I would more like to stay with this core tests style of creating channel args.